packages/phoenix-otel/src/phoenix/otel/otel.py

```diff
@@ -78,80 +94,107 @@ def __init__(self, *args: Any, endpoint: Optional[str] = None, **kwargs: Any):
-        parsed_url = urlparse(endpoint)
-        assert isinstance(parsed_url, ParseResult)
+        parsed_url, endpoint = _normalized_endpoint(endpoint)
         self._default_processor = False
 
         if _maybe_http_endpoint(parsed_url):
-            print("Exporting spans via HTTP.")
             http_exporter: SpanExporter = HTTPSpanExporter(endpoint=endpoint)
-            self.add_span_processor(SimpleSpanProcessor(exporter=http_exporter))
+            self.add_span_processor(SimpleSpanProcessor(span_exporter=http_exporter))
             self._default_processor = True
         elif _maybe_grpc_endpoint(parsed_url):
-            print("Exporting spans via GRPC.")
             grpc_exporter: SpanExporter = GRPCSpanExporter(endpoint=endpoint)
-            self.add_span_processor(SimpleSpanProcessor(exporter=grpc_exporter))
+            self.add_span_processor(SimpleSpanProcessor(span_exporter=grpc_exporter))
             self._default_processor = True
-        else:
-            warnings.warn(
-                "Could not infer exporter to use. Use `add_span_processor` to configure span "
-                "processing and export."
-            )
+        if verbose:
+            print(self._tracing_details())
 
     def add_span_processor(self, *args: Any, **kwargs: Any) -> None:
         if self._default_processor:
-            print("Overriding default span processor.")
             self._active_span_processor.shutdown()
             self._active_span_processor._span_processors = tuple()  # remove default processors
             self._default_processor = False
         return super().add_span_processor(*args, **kwargs)
 
+    def _tracing_details(self) -> str:
+        project = self.resource.attributes.get(PROJECT_NAME)
+        processor_name: Optional[str] = None
+        endpoint: Optional[str] = None
+        transport: Optional[str] = None
+        headers: Optional[Union[Dict[str, str], str]] = None
+
+        if self._active_span_processor:
+            if processors := self._active_span_processor._span_processors:
+                if len(processors) == 1:
+                    span_processor = self._active_span_processor._span_processors[0]
+                    if exporter := getattr(span_processor, "span_exporter"):
+                        processor_name = span_processor.__class__.__name__
+                        endpoint = exporter._endpoint
+                        transport = _exporter_transport(exporter)
+                        headers = _normalize_headers(exporter._headers)
+                else:
+                    processor_name = "Multiple Span Processors"
+                    endpoint = "Multiple Span Exporters"
+                    transport = "Multiple Span Exporters"
+                    headers = "Multiple Span Exporters"
+
+        if os.name == "nt":
+            details_header = "OpenTelemetry Tracing Details"
+        else:
+            details_header = "🔭 OpenTelemetry Tracing Details 🔭"
```

Comment:
should the guard be just for windows?

Reply:
The current guard is only for Windows, are we worried about anything else? (cc @RogerHYang)